code_samples/front/view_matcher/config/packages/views.yaml

```diff
@@ -12,4 +12,5 @@ ibexa:
                         template: '@ibexadesign/full/featured_article.html.twig'
                         match:
                             Identifier\ContentType: article
-                            \App\View\Matcher\Owner: [johndoe, janedoe]
+                            #\App\View\Matcher\Owner: [johndoe, janedoe]
```

Comment:
I would skip even mentioning class-based notation. It's misleading, because you can't use custom constructor for such matcher. I'd like to get rid of class-based notation entirely in 5.0.0 if possible. Or at least deprecate it at some point.

Reply:
FQCN removed in 979e0f21adae56518d5e66f8a612c15eeeabfbbf